samtools/1.7.0/Dockerfile

```diff
@@ -0,0 +1,28 @@
+################## BASE IMAGE ######################
+
+FROM biocontainers/biocontainers:latest
+
+################## METADATA ######################
+
+ARG software_version=1.7.0
+LABEL base_image="biocontainers:latest"
+LABEL version="2"
+LABEL software="Samtools"
```

Comment:
software name must match regexp [a-z0-9_-] (lowercase) as name is used for container naming and docker names have restrictions